sources/packages/backend/apps/api/src/route-controllers/application/models/application.dto.ts

```diff
@@ -142,9 +142,14 @@ export class ApplicationSupportingUserDetails {
   partnerInfo?: SuccessWaitingStatus;
 }
 
+export class ApplicationInCalculationDetails {
```

Comment:
in this case, I would recommend to just add one more property to `ApplicationProgressDetailsAPIOutDTO`

like this

```ts
export class InProgressApplicationDetailsAPIOutDTO extends IntersectionType(
  ApplicationSupportingUserDetails,
  ApplicationIncomeVerification,
  ApplicationInCalculationDetails,
) {
  id: number;
  applicationStatus: ApplicationStatus;
  pirStatus: ProgramInfoStatus;
  pirDeniedReason?: string;
  exceptionStatus?: ApplicationExceptionStatus;
  outStandingAssessmentStatus: SuccessWaitingStatus;
}
```